packages/sdk/src/across/clients/bridgePool.ts

```diff
@@ -6,7 +6,8 @@ import Multicall2 from "../../multicall2";
 import TransactionManager from "../transactionManager";
 import { ethers, Signer, BigNumber } from "ethers";
 import type { Overrides } from "@ethersproject/contracts";
-import { TransactionRequest, TransactionReceipt, Provider, Log, Block } from "@ethersproject/abstract-provider";
+import { TransactionRequest, TransactionReceipt, Log } from "@ethersproject/abstract-provider";
+import { Provider, Block } from "@ethersproject/providers";
```

Comment:
I changed the source of the imported types in order to avoid some conflicts between the installed packages. The types generated by `typechain` use `@ethersproject/providers`, not `"@ethersproject/abstract-provider"`